test/systemtests/util_test.go

```diff
@@ -1375,6 +1375,16 @@ func (s *systemtestSuite) SetUpTestBaremetal(c *C) {
 			PvtSubnet:        "172.19.0.0/16",
 		}), IsNil)
 		time.Sleep(40 * time.Second)
+	} else {
+		c.Assert(s.cli.GlobalPost(&client.Global{FwdMode: "bridge",
+			Name:             "global",
+			NetworkInfraType: "default",
+			Vlans:            "1-4094",
+			Vxlans:           "1-10000",
+			ArpMode:          "proxy",
+			PvtSubnet:        "172.19.0.0/16",
+		}), IsNil)
+		time.Sleep(40 * time.Second)
```

Comment:
since everything but the FwdMode string is the same, just make that a variable set via an if else and pass in the variable?

Reply:
done